modules/storage/storage-account/main.bicep

```diff
@@ -0,0 +1,56 @@
+@description('Deployment Location')
+param location string
```

Comment:
Was there a particular reason why `location` is required?

Reply:
yes. When we provide a default value for location, it causes an ARM-TTK issue when submitting to the Azure Marketplace.
https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/linter-rule-no-hardcoded-location

Reply:
I see - I'm good leaving it as is for now, but I think it might be worth revisiting this in the future.